### PR TITLE
added url shortener

### PR DESCRIPTION
**Pull request by baljinderxd**
For #81 

`graph 180` working 🎉 

image

### CODE REVIEW

File: app/commands/graph.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const prepareAnswer = require('../utils/prepareAnswer');
 
 function graph(body, bot, chatId, n = 10) {
-  const data = prepareAnswer.summaryGraph(body, n);
+  prepareAnswer.summaryGraph(body, n, chatId, bot);
```
Comment (gurrrung) [inline app/commands/graph.js:4]: `summaryGraph` should return the URL.

File: app/commands/graph.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
-  return bot.sendPhoto(chatId, data);
+  return true;
```
Comment (gurrrung) [inline app/commands/graph.js:6]: `sendPhoto` call needs to be here.

File: app/utils/prepareAnswer.js [hunk 3]
```diff
@@ -347,10 +347,11 @@ function dailyDecStatsGraph(body, n) {
   return link;
 }
 
-function summaryGraph(body, n) {
+const QuickChart = require('quickchart-js');
```
Comment (gurrrung) [inline app/utils/prepareAnswer.js:350]: Please load the module always at the beginning of the file.

File: app/utils/prepareAnswer.js [hunk 4]
```diff
@@ -347,10 +347,11 @@ function dailyDecStatsGraph(body, n) {
...
+
+function summaryGraph(body, n, chatId, bot) {
```
Comment (gurrrung) [inline app/utils/prepareAnswer.js:352]: this function can be `async`.

File: app/utils/prepareAnswer.js [hunk 5]
```diff
@@ -362,21 +363,46 @@ function summaryGraph(body, n) {
...
+      },
+      options: {
+        title: {
+          display: true,
+          text: `Daily cases analysis for past ${minN} days`,
+        },
+      },
+    }).setDevicePixelRatio(2);
+
+  async function gaphWithShortUrl() {
```
Comment (gurrrung) [inline app/utils/prepareAnswer.js:401]: You don't need this another function just use `await` here.

**Review by gurrrung — CHANGES_REQUESTED** [review]
Good job! Thanks for your contribution. I have a few change requests.

**Comment by baljinderxd** [comment]
Sorry, I forgot to run the lint fix xD. Will correct it and push a new commit in 15 minutes😬

File: app/utils/prepareAnswer.js (hunk 5 above)
Comment (baljinderxd) [reply app/utils/prepareAnswer.js]: Okay I removed this extra function 👍

File: app/utils/prepareAnswer.js (hunk 4 above)
Comment (baljinderxd) [reply app/utils/prepareAnswer.js]: Done 😬

File: app/utils/prepareAnswer.js (hunk 3 above)
Comment (baljinderxd) [reply app/utils/prepareAnswer.js]: Done💯

File: app/commands/graph.js (hunk 1 above)
Comment (baljinderxd) [reply app/commands/graph.js]: Implemented 🙌🙌

File: app/commands/graph.js (hunk 2 above)
Comment (baljinderxd) [reply app/commands/graph.js]: Did this too 💯

**Review by gurrrung — APPROVED** [review]
LGTM

**Pull request merged**